test/libsolidity/semanticTests/enums/enum_explicit_overflow.sol

```diff
@@ -9,12 +9,12 @@ contract test {
     }
 
     function getChoiceFromSigned(int256 x) public returns (uint256 d) {
-        choice = ActionChoices(x);
+        choice = ActionChoices(type(uint).max);
         d = uint256(choice);
     }
 
     function getChoiceFromNegativeLiteral() public returns (uint256 d) {
-        choice = ActionChoices(-1);
+        choice = ActionChoices(type(uint).max);
```

Comment:
The name should be updated because it's no longer a literal (it would be a compilation error otherwise).

Same in the identical test for homestead.